### PR TITLE
Test: merkle ganache accounts

### PR DESCRIPTION
**Pull request by hanszhoou**

### CODE REVIEW

File: src/generate.ts [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
-  const leaves = ${JSON.stringify(leavesWithProof, null, 2)}
-
-  module.exports = {
-    merkleRoot,
-    leaves
-  }
-` +
+  return (
+    endent`
+    const merkleRoot = "${tree.root}"
```
Comment (septs) [inline src/generate.ts]:
```suggestion
    const merkleRoot = ${JSON.stringify(tree.root)}
```

File: src/generate.ts [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
+    endent`
+    const merkleRoot = ${JSON.stringify(tree.root)}
+    const leaves = ${JSON.stringify(leavesWithProof, null, 2)}
+  
+    module.exports = {
+      merkleRoot,
+      leaves
+    }
+  ` +
+    `
```
Comment (septs) [inline src/generate.ts]:
```suggestion
  return 'module.exports = ' + JSON.stringify({ merkleRoot, leaves }, null, 2)
```

File: test.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { promises as fs } from 'fs'
+import ganache from 'ganache-core'
+import { JsonRpcResponse, JsonRpcPayload } from 'ganache-core'
+import { exec, ExecException } from 'child_process'
```
Comment (septs) [inline test.ts:4]:
```suggestion
import { spawn, ExecException } from 'child_process'
```

File: test.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  provider.send({ method: 'eth_accounts' } as JsonRpcPayload, async (err: Error | null, response?: JsonRpcResponse) => {
+    if (!response) {
+      server.close()
+      throw new Error('🚨 no accounts')
+    }
+    const template = generate(response.result)
+    await fs.writeFile('./test/constants.js', template)
+    console.log('✨ test/constants.js generated')
+
+    exec('truffle test', (err: ExecException | null, stdout: string, _stderr: string) => {
```
Comment (septs) [inline test.ts:24]:
```suggestion
    spawn('truffle test', (err: ExecException | null, stdout: string, _stderr: string) => {
```

File: contracts/airdrop.sol [hunk 5]
```diff
@@ -62,7 +62,11 @@ contract Airdrop {
         if (!available) {
             claimable = 0;
         } else if (block.timestamp > info.start_time) {
-            claimable = amount - amount * (((block.timestamp - info.start_time) / 86400) * 2) / 10;
+            if ((block.timestamp - info.start_time) / 86400 > 5) {
```
Comment (hanszhoou) [inline contracts/airdrop.sol:65]: SafeMath doesn't take effect, after 6 days after start_time, overflow.

```
should decrease the amount by 100% after six days after start_time:
AssertionError: expected '115792089237316195423570985008687907853269984665640564039457584007913129631827' to equal '0'
```

Comment (yisiliu) [reply contracts/airdrop.sol:65]: Since here info.start_time is uint48. I will fix this.

**Pull request merged**